src/vzlogger.cpp

```diff
@@ -549,6 +585,15 @@ int main(int argc, char *argv[]) {
 					::sleep(1); // 1s should be ok. will introduce a shutdown latency >1s
 				}
 			}
+			if (mainLoopReopenLogfile) {
+				mainLoopReopenLogfile = false;
+				print(log_info, "closing logfile for re-opening (requested with SIGUSR1)", "");
```

Comment:
@Zugschlus:
i guess i should be passing NULL instead of `""` to avoid the empty pair of breackets in the log message.
feel free to open an MR to change it...
https://github.com/volkszaehler/vzlogger/blob/master/src/vzlogger.cpp#L168